amazon-ssm-agent.json

```diff
@@ -0,0 +1,45 @@
+{
```

Comment:
this is the default SSM agent config file, from here:
https://github.com/aws/aws-codebuild-docker-images/blob/master/ubuntu/standard/6.0/amazon-ssm-agent.json